doc/user/content/sql/functions/now_and_mz_now.md

```diff
@@ -49,26 +53,18 @@ a logical timestamp within a few seconds of 9pm, even if data for 8:30–9pm has
 not yet arrived and the query will need to block until the data for 9pm arrives.
 In this scenario, both `now()` and `mz_now()` would return 9pm.
 
-### Limitations
+### Materialization limitations
 
-#### Materialization
-  * Queries that use `now()` cannot be materialized. In other words, you cannot
-    create an index or a materialized view on a query that calls `now()`.
+* Queries that use `now()` cannot be materialized. In other words, you cannot
+  create an index or a materialized view on a query that calls `now()`.
 
-  * Queries that use `mz_now()` can only be materialized if the call to
-    `mz_now()` is used in a [temporal filter](/sql/patterns/temporal-filters).
+* Queries that use `mz_now()` can only be materialized if the call to
+  `mz_now()` is used in a [temporal filter](/sql/patterns/temporal-filters).
 
 These limitations are in place because `now()` changes every microsecond and
 `mz_now()` changes every millisecond. Allowing these functions to be
 materialized would be resource prohibitive.
 
-#### Comparison operators
```

Comment:
Moved this into the new mz_now clause section:
https://github.com/MaterializeInc/materialize/pull/33041/files#diff-3c8242dd8b8ccceef1b8650d257e7a9e38ed2cc964839b6f0f58497146340661R19